timon/resources.py

```diff
@@ -19,7 +22,7 @@ class TiMonResource():
     def __init__(self, name, count):
         self.name = name
         self.count = count
-        self.semaph = Semaphore(count)
+        self.semaph = Semaphore(count)  # TODO: maybe add max_value too
```

Comment:
what do you mean?
count is the max value, no?

Reply:
seriosly I'm not convinced this is really useful. 

but if you feel safer limiting the max value of the semaphore.

then just use asyncio.BoundedSemaphore instead of asyncio.Semaphore

Reply:
I removed the comment